validation-framework/secrets/credential.json

```diff
@@ -0,0 +1,4 @@
+{ 
+	"email": "cc_onboard+v2@ccloud.com", 
+	"password": "hkbcb3rRcL7$Qjyx5pV@X%7^p#2@o6"
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Medium

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

 <!-- a0ce3514-e69f-4e19-bbfa-cc5b3ae4d2af -->